rest/servers/spring/src/main/java/com/connexta/ingest/rest/spring/IngestApi.java

```diff
@@ -84,11 +84,11 @@
               OffsetDateTime lastModified,
       @ApiParam(
               value =
-                  "A file attachment that is sent in the ingest request. The current maximum is the character equivalent of 10GB.",
+                  "A product attachment that is sent in the ingest request. The current maximum is the character equivalent of 10GB.",
```

Comment:
Is this right? To me file was correct and product is a processed file that was marked ready for dissemination.

Reply:
Thanks. The Architects changed the definitions again after I pushed this last commit. I'll change it back to `file` instead of `project`.